src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -198,7 +198,7 @@ class SyncController(
       else
         (SyncState.empty, fastSyncStateStorage.purge())
 
-    private var mptNodesQueue: Set[HashType] = initialSyncState.mptNodesQueue
+    private var mptNodesQueue: Seq[HashType] = initialSyncState.mptNodesQueue
     private var nonMptNodesQueue: Set[HashType] = initialSyncState.nonMptNodesQueue
     private var blockBodiesQueue: Set[ByteString] = initialSyncState.blockBodiesQueue
     private var receiptsQueue: Set[ByteString] = initialSyncState.receiptsQueue
```

Comment:
> Why not change all of them?

order is not required in case of storage root and evm code as evm code does not expand to new hashes and mpt root expands to branches that are later added to `Seq`